java/Oleksii-Riabov/src/com/zoolatech/lecture5/tasks/_3/Task3.java

```diff
@@ -0,0 +1,28 @@
+package com.zoolatech.lecture5.tasks._3;
+
+import java.util.List;
+
+/**
+ * Create a method that accepts a list of words, and displays only
+ * unique one (order can be random). Bonus task: treat strings in a
+ * different case as the same one: “Apple” and “apple” should be
+ * treated as the same string (don’t use any methods that create a
+ * new string object - like toUpperCase or toLowerCase - you can
+ * work only with a given input).
+ * Input: [“apple”, “orange”, “pear”, “apple”, “banana”, “orange”]
+ * Output: apple
+ *             orange
+ *             pear
+ *             banana
+ */
+
+public class Task3 {
+
+    public static void main(String[] args) {
+        displayUniqueWords(List.of("apple", "orange", "pear", "Apple", "banana", "orange"));
+    }
+
+    public static void displayUniqueWords(List<String> list) {
+        list.stream().map(String::toLowerCase).distinct().forEach(System.out::println);
```

Comment:
```
don’t use any methods that create a new string object - like toUpperCase or toLowerCase - you can work only with a given input
```

Reply:
Task has been changed to meet this requirement.